anaconda-pkg-build/linux/rocky8/Dockerfile

```
@@ -61,8 +61,11 @@ RUN MC_ARCH="$(uname -m)" \
     && /opt/conda/bin/conda install --quiet --yes conda-build \
     && /opt/conda/bin/conda clean --all --yes \
     && rm -fv /opt/conda/.condarc \
+    && /opt/conda/bin/conda config --system --add channels defaults \
     && /opt/conda/bin/conda config --system --set show_channel_urls True \
     && /opt/conda/bin/conda config --system --set add_pip_as_python_dependency False \
+    && /opt/conda/bin/conda config --system --set report_errors False \
+    && /opt/conda/bin/conda config --system --set conda_build.pkg_format ".tar.bz2" \
```

```suggestion
    && /opt/conda/bin/conda config --system --set conda_build.pkg_format 1 \
```
https://github.com/conda/conda-build/blob/25.5.0/CHANGELOG.md#enhancements-5
The doc says to use 1.